avalaratax.php

```diff
@@ -1635,7 +1653,9 @@ public function getTax($products = array(), $params = array())
         // Attempt to obtain and set customer entity use code for this request
         $entity_use_code = $this->getEntityUseCode($customerCode); // $customerCode should be their customer_id
 
-        if (!empty(trim($entity_use_code))) {
+        # PHP less than 5.5.0 cannot use expressions in empty()
+        # e.g. if(!empty(trim($entity_use_code)))
+        if (!(trim($entity_use_code) == false)) {
```

Comment:
PHP < 5.5 cannot execute expressions in empty(), but you can separate your calls in order to make it work.

i.e:
```php
$trim_entry_use_code = trim($entity_use_code);
if (!empty($trim_entity_use_code)) {
    // blabla
```

By doing this, you keep all the functionalities you expected.